### ISSUE TITLE
App sometimes crashes

### ISSUE THREAD

**Issue by mjain234** [KieronQuinn/ClassicPowerMenu#54]
Hi there
I have using Classic Power Menu for a while but suddenly the app crashing frequently but sometimes it works l, don't know why this is happening now.

Check this log 
https://stagbin.tk/zsyjbzl

**Comment by KieronQuinn** [comment]
```
time: 1656954988747
msg: org.koin.core.error.NoBeanDefFoundException: |- No definition found for class:'com.android.systemui.broadcast.BroadcastDispatcher'. Check your definitions!
stacktrace: java.lang.RuntimeException: Unable to instantiate activity ComponentInfo{com.kieronquinn.app.classicpowermenu/com.android.systemui.controls.management.ControlsProviderSelectorActivity}: org.koin.core.error.NoBeanDefFoundException: |- No definition found for class:'com.android.systemui.broadcast.BroadcastDispatcher'. Check your definitions!
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3613)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3864)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2253)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7866)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:550)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
Caused by: org.koin.core.error.NoBeanDefFoundException: |- No definition found for class:'com.android.systemui.broadcast.BroadcastDispatcher'. Check your definitions!
	at org.koin.core.scope.Scope.throwDefinitionNotFound(Scope.kt:304)
	at org.koin.core.scope.Scope.resolveValue(Scope.kt:274)
	at org.koin.core.scope.Scope.resolveInstance(Scope.kt:241)
	at org.koin.core.scope.Scope.get(Scope.kt:204)
	at com.android.systemui.controls.management.ControlsProviderSelectorActivity$special$$inlined$inject$default$4.invoke(ComponentCallbackExt.kt:61)
	at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
	at com.android.systemui.controls.management.ControlsProviderSelectorActivity.getBroadcastDispatcher(ControlsProviderSelectorActivity.kt:54)
	at com.android.systemui.controls.management.ControlsProviderSelectorActivity.<init>(ControlsProviderSelectorActivity.kt:63)
	at java.lang.Class.newInstance(Native Method)
	at android.app.AppComponentFactory.instantiateActivity(AppComponentFactory.java:95)
	at androidx.core.app.CoreComponentFactory.instantiateActivity(CoreComponentFactory.java:45)
	at android.app.Instrumentation.newActivity(Instrumentation.java:1291)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3600)
	... 12 more

```

**Comment by KieronQuinn** [comment]
Should be fixed in 1.6.
